sdk/vaa/structs.go

```diff
@@ -645,21 +645,42 @@ func (v *BatchVAA) signingBody() []byte {
 	return buf.Bytes()
 }
 
-// SigningMsg returns the hash of the signing body.
```

Comment:
since this is part of the SDK, could this interface change break users of the sdk and would require a major version increase? Maybe just better to keep the deprecated method for now and just mark it as deprecated instead of removing it